Assets/Scripts/Details/QMDetailsController.cs

```diff
@@ -27,14 +27,16 @@ public class QMDetailsController : MonoBehaviour
     // TEST: The Button used to trigger the window
     public Button testButton;
     
-    void Clicked(){
+    public void Clicked(){
         ClearAllInfoFields();
         QMDetailsWindow.SetActive(true);
         QueueManagerInfoInit(selectedQM);
     }
 
     void Start()
-    {
+    {
+        // TODO: DEMO TEST
+        Queue.tempWindow = this;
```

Comment:
there are many TODO comments to remove `tempWindow` variable, what's the plan with it and why to keep it?

Reply:
> there are many TODO comments to remove `tempWindow` variable, what's the plan with it and why to keep it?

As indicated during our meetings, at the time of development there exist(s) no proper functionality for rendering specific Queue windows. We do not intend to show QM details in this way, as such this functionality was to serve as a DEMO of how this would work. As discussed, further iterations derived from #28 should implement this functionality.
The TODOs serve as notice point.

Reply:
yeah, that makes sense